Demo/src/main/java/com/braintreepayments/demo/PayPalFragment.java

```diff
@@ -91,6 +187,18 @@ private void clearPendingRequest() {
         PendingRequestStore.getInstance().clearPayPalPendingRequest(requireContext());
     }
 
+    private void storeEditFiPendingRequest(PayPalVaultEditPendingRequest.Started request) {
+        PendingRequestStore.getInstance().putPayPalPendingRequestEditFi(requireContext(), request);
+    }
+
+    private PayPalVaultEditPendingRequest.Started getPendingRequestEditFi() {
+        return PendingRequestStore.getInstance().getPayPalPendingRequestEditFi(requireContext());
+    }
+
+    private void clearEditFiPendingRequest() {
```

Comment:
Can we remove Fi from all of our naming since it isn't used in any of the public class names? Maybe just `PayPalVaultEdit` for the prefix on everything